css/index.css

```diff
@@ -104,7 +147,105 @@ p {
 .middle-content .boxes .box {
```

Comment:
Great job figuring this out with nth child selector! I'd recommend using different class names for it, though!